src/App.tsx

```diff
@@ -1,54 +1,89 @@
-import React from 'react';
+import React, { useState, useMemo, useCallback } from 'react';
 import './App.scss';
```

Comment:
according to the style guide, import styles should be at the bottom of the imports list